FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta1/ota/common/DemoTasks/aws_iot_ota_update_demo.c

```diff
@@ -328,7 +326,7 @@ OTA_Err_t xErr = kOTA_Err_Uninitialized;
 		/* We should never get here as new image activation must reset the device.*/
 		for( ; ; )
 		{
-			__debugbreak();
+			IotClock_SleepMs( otaDemoTASK_DELAY_SECONDS * 1000 );
```

Comment:
Should be logging a warning message here since we're not supposed to get here?

Reply:
on line 324, the function `OTA_ActivateNewImage` is taking care of that, on failure it will print "Error: Failed to activate new image, Error code - (0x%08x). Please reset manually."

Reply:
Using `configPRINTF( ( "Error! New image activation failed.\r\n" ) );`  should fix build checks